diff --git a/scripts/msmtpq/msmtpq b/scripts/msmtpq/msmtpq
index d8b4039..12eff8e 100755
--- a/scripts/msmtpq/msmtpq
+++ b/scripts/msmtpq/msmtpq
@@ -149,19 +149,19 @@ log() {
 
   [ -z "$EMAIL_QUEUE_QUIET" ] && dsp "$@"  # display msg to user, as well as logging it
 
-  if [ -n "$LOG" ] ; then            # log is defined and in use
-    for ARG ; do                     # each msg line out
-      [ -n "$ARG" ] && \
-        echo "$PFX : $ARG" >> "$LOG" # line has content ; send it to log
-    done
-  fi
-
-  if [ -n "$RC" ] ; then             # an error ; leave w/error return
-    [ -n "$LKD" ] && lock_queue -u   # unlock here (if locked)
-    [ -n "$LOG" ] && \
-      echo "    exit code = $RC" >> "$LOG" # logging ok ; send exit code to log
-    exit "$RC"                       # exit w/return code
-  fi
+  # if [ -n "$LOG" ] ; then            # log is defined and in use
+  #   for ARG ; do                     # each msg line out
+  #     [ -n "$ARG" ] && \
+  #       echo "$PFX : $ARG" >> "$LOG" # line has content ; send it to log
+  #   done
+  # fi
+
+  # if [ -n "$RC" ] ; then             # an error ; leave w/error return
+  #   [ -n "$LKD" ] && lock_queue -u   # unlock here (if locked)
+  #   [ -n "$LOG" ] && \
+  #     echo "    exit code = $RC" >> "$LOG" # logging ok ; send exit code to log
+  #   exit "$RC"                       # exit w/return code
+  # fi
 }
 
 ## write/remove queue lockfile for a queue op
